config.go

```diff
@@ -271,6 +288,16 @@ func loadConfig() (*config, error) {
 		Alias:        defaultAlias,
 		Color:        defaultColor,
 		MinChanSize:  int64(minChanFundingSize),
+		Tor: &torConfig{
+			SOCKS:   defaultTorSOCKS,
+			DNS:     defaultTorDNS,
+			Control: defaultTorControl,
+			// TODO: change this to v3 once we're able to
+			// automatically set them up.
+			V2:               true,
```

Comment:
not sure if it's obvious that this will be `true` by default. Looks like the default is not shown in `lnd -h`

Reply:
Fixed.